### PR TITLE
[Testing] Imagery Plugin Tests

### LINKED ISSUE #3328: [Testing] Create tests for Imagery Plugin
Imagery Plugin currently has no test specs, need to create a pluginSpec.js.

## Testing
- Go into the Imagery Plugin folder and alter the test spec's first "describe" function as an "fdescribe"
- Run tests and ensure no errors

### PR DESCRIPTION
**Pull request by jvigliotta**
Created tests for the imagery plugin. 

Testing includes: 
- imagery view provider (accepts image telemetry producing objects)
- shows most recent image on load
- shows correct image on thumnail click
- shows newness indicator correctly based on imagery age

### Author Checklist
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | Y
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

closes #3328

### CODE REVIEW

File: src/utils/testing.js [hunk 1]
```diff
@@ -90,6 +90,26 @@ export function resetApplicationState(openmct) {
     return promise;
 }
 
+// required: key
+// optional: element, keyCode, type
+export function simulateKeyEvent(opts) {
+
+    if (!opts.key) {
+        console.warn('simulateKeyEvent needs a key');
```
Comment (nikhilmandlik) [inline src/utils/testing.js:98]: may be return as well?

File: src/utils/testing.js [hunk 2]
```diff
@@ -90,6 +90,26 @@ export function resetApplicationState(openmct) {
...
+    }
+
+    let el = opts.element || document;
```
Comment (nikhilmandlik) [inline src/utils/testing.js:101]: `let` -> `const` 😄

**Review by nikhilmandlik — COMMENTED** [review]
Small changes

File: src/utils/testing.js (hunk 1 above)
Comment (jvigliotta) [reply src/utils/testing.js:98]: got it

Comment (jvigliotta) [reply src/utils/testing.js:98]: got it

File: src/utils/testing.js (hunk 2 above)
Comment (jvigliotta) [reply src/utils/testing.js]: changed em all!!!!!! muwhahahahaha

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Pull request merged**